pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -75,7 +75,16 @@ func (io *BootstrapParameters) Run() error {
 	if err != nil {
 		return err
 	}
-	log.Success("Bootstrapped GitOps sucessfully.")
```

Comment:
We probably only need to link to one URL.

Also, this should not be duplicated, let's extract it to a separate function?

Reply:
sure, I guess boottrap command will be completely removed so just to test the output I kept it there. Are we keeping both bootstrap and wizard command?